manifests/init.pp

```diff
@@ -1,208 +1,132 @@
-  }
-
-  $real_agent_utils = $agent_utils ? {
-    'UNSET' => $::teagent::params::agent_utils,
-    default => $agent_utils,
-  }
-
-  $real_international_langs = $international_langs ? {
-    'UNSET' => $::teagent::params::international_langs,
-    default => $international_langs,
-  }
-
-  $real_account_token = $account_token ? {
-    'UNSET' => $::teagent::params::account_token,
-    default => $account_token,
-  }
-
-  $real_log_path = $log_path ? {
-    'UNSET' => $::teagent::params::log_path,
-    default => $log_path,
-  }
-
-  $real_log_file_size = $log_file_size ? {
-    'UNSET' => $::teagent::params::log_file_size,
-    default => $log_file_size,
-  }
-
-  $real_proxy_host = $proxy_host ? {
-    'UNSET' => $::teagent::params::proxy_host,
-    default => $proxy_host,
-  }
-
-  $real_proxy_port = $proxy_port ? {
-    'UNSET' => $::teagent::params::proxy_port,
-    default => $proxy_port,
-  }
-
-  $real_proxy_user = $proxy_user ? {
-    'UNSET' => $::teagent::params::proxy_user,
-    default => $proxy_user,
-  }
-
-  $real_proxy_pass = $proxy_pass ? {
-    'UNSET' => $::teagent::params::proxy_pass,
-    default => $proxy_pass,
-  }
-
-  $real_proxy_bypass_list = $proxy_bypass_list ? {
-    'UNSET' => $::teagent::params::proxy_bypass_list,
-    default => $proxy_bypass_list,
-  }
-
-  $real_ip_version = $ip_version ? {
-    'UNSET' => $::teagent::params::ip_version,
-    default => $ip_version,
-  }
-
-  $real_interface = $interface ? {
-    'UNSET' => $::teagent::params::interface,
-    default => $interface,
-  }
-
-  $real_set_repo = $set_repo ? {
-    'UNSET' => $::teagent::params::set_repo,
-    default => $set_repo,
-  }
-
-  # this takes care of the dependencies (repos) if set to true
-  if ($real_set_repo) {
-    include teagent::dependency
-    $teagent_requires = [
-      Class['teagent::dependency'],
-      Class['teagent::repository'],
-    ]
-  }
-  else {
-    $teagent_requires = []
-  }
-
-  package { 'te-agent':
-    ensure  => 'installed',
-    require => $teagent_requires,
-  }
-
-  if $real_browserbot {
-    package { 'te-browserbot':
-      ensure  => 'installed',
-      require => Package['te-agent'],
-    }
-  }
-
-  if $real_agent_utils {
-    package { 'te-agent-utils':
-      ensure  => 'latest',
-      require => Package['te-agent'],
-    }
-  }
-
-  if $real_international_langs {
-    package { 'te-intl-fonts':
-      ensure  => 'latest',
-      require => [
-        Class['teagent::dependency'],
-        Class['teagent::repository'],
-      ],
-    }
-  }
-
-  file { '/var/lib/te-agent/config_teagent.sh':
-    ensure  => 'present',
-    content => template('teagent/config_teagent.sh.erb'),
-    mode    => '0755',
-    require => [ Package['te-agent'], ],
-  }
-
-  exec { '/var/lib/te-agent/config_teagent.sh':
-    subscribe   => File['/var/lib/te-agent/config_teagent.sh'],
-    refreshonly => true,
-  }
+# Copyright © 2017 ThousandEyes, Inc.
+#
+
+class te_agent(
+Optional[String] $proxy_user = undef,
```

Comment:
This parameters are missing the indentation right? At least they need 2 spaces, and also, can you align the `=`?